app/src/main/res/values/strings.xml

```diff
@@ -260,4 +260,6 @@
     <string name="fxa_settings_primary_button">%1$s tabs</string>
     <!-- Button text to sign out from the Firefox Accounts settings page -->
     <string name="fxa_settings_secondary_button">Sign out</string>
+    <!-- This is a content description for an image that will display the avatar of the currently logged in user -->
```

Comment:
nit: localizers probably don't know what a "content description" is: can you explain: "this text is announced to screen reader users when they focus..."

That being said, can this image even be focused? Is a content description necessary?

Reply:
IMO it's a good approach to add a content description to every image rather than risk something getting missed.  Especially if focusability ever changes.

I will make that copy change though.